src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -14,240 +14,227 @@
  */
 public class HtmlTitleTests extends NewTestTemplate {
 
+  // TODO: QAART-681 Sonar should not report duplicate strings warnings for data providers
+  private static final String TEST_WIKI_CORPORATE_FR = "fr";
+  private static final String TEST_WIKI_CORPORATE_PL = "pl";
+  private static final String TEST_WIKI_CORPORATE = "www";
   private static final String TEST_WIKI_CUSTOM_TITLE = "es.pokemon";
   private static final String TEST_WIKI_ORIGINAL_TITLE = "sktest123";
+  private static final String TEST_WIKI_ORIGINAL_TITLE_WITH_EM_DASH = "poznan";
 
   Credentials credentials = Configuration.getCredentials();
 
   @DataProvider
-  private Object[][] dataHtmlTitleTest001() {
+  private Object[][] dataHtmlTitleTest() {
     return new Object[][]{
-        // original title
+        // Original title
+        // TODO: SEO-77 Fix the title of the main pages for Mercury
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)